Module/Module.cc

```diff
@@ -16,24 +16,41 @@
 HANDLE simconnect = 0;
 
 void CALLBACK HandleSimconnectMessage(SIMCONNECT_RECV* pData, DWORD cbData, void* pContext) {
-  switch (pData->dwID) {
-    case SIMCONNECT_RECV_ID_CLIENT_DATA: {
-      auto received_data = static_cast<SIMCONNECT_RECV_CLIENT_DATA*>(pData);
-      jetbridge::Packet* packet = (jetbridge::Packet*)&received_data->dwData;
+  if (pData->dwID != SIMCONNECT_RECV_ID_CLIENT_DATA) return;
 
-      // TODO: better opcode decoding.
-      if (packet->data[0] != 'x') return;
+  auto received_data = static_cast<SIMCONNECT_RECV_CLIENT_DATA*>(pData);
+  jetbridge::Packet* packet = (jetbridge::Packet*)&received_data->dwData;
 
-      std::string code = (char*)(packet->data + 1);
-      std::cout << "[Jetbridge] [" << packet->id << ":x] " << code << std::endl;
-      execute_calculator_code(code.c_str(), 0, 0, 0);
+  // We'll prepare our response packet with the same id.
+  auto response = new jetbridge::Packet();
+  response->id = packet->id;
 
-      // Reply with the same exact same packet (for now).
-      SimConnect_SetClientData(simconnect, jetbridge::kPublicDownlinkArea, jetbridge::kPacketDefinition, 0, 0,
-                               sizeof(jetbridge::Packet), packet);
+  // Split packet data to opcode and payload.
+  char opcode = packet->data[0];
+  std::string payload = (char*)(packet->data + 1);
+
+  switch (opcode) {
+    case jetbridge::kExecuteCalculatorCode: {
+      execute_calculator_code(payload.c_str(), 0, 0, 0);
+      break;
+    }
+
+    case jetbridge::kGetNamedVariable: {
+      int named_variable = check_named_variable(payload.c_str());

```

Comment:
Might be worth not replying (i.e. just doing `return`) if the variable was not found (i.e. `named_variable == -1`)